platform/ios/src/MGLMapView.mm

```diff
@@ -1128,9 +1128,68 @@ - (void)wakeGL:(__unused NSNotification *)notification
         [self validateLocationServices];
 
         [MGLMapboxEvents pushEvent:MGLEventTypeMapLoad withAttributes:@{}];
+        
+        [self updateLocalization];
```

Comment:
If the developer switches to a new style while the application is already in the foreground, the labels won’t be localized until the user switches away and back to the application. Instead, we should localize the labels in response to `-mapView:didFinishLoadingStyle:`, [as MapDocument does](https://github.com/mapbox/mapbox-gl-native/blob/f053feb6d4aba1eaa44a35d050522b641c011b33/platform/macos/app/MapDocument.m#L1108-L1110) in macosapp.

Additionally, if the developer adds a named symbol layer after loading the style, using the runtime styling API, we should localize it consistently. If we’re concerned about that possibility, we could require a change to either `-[MGLSymbolStyleLayer setText:]` or `-[MGLStyle addLayer:]`.